examples/hello_world/tests/harness.rs

```diff
@@ -0,0 +1,43 @@
+use fuel_tx::Salt;
+use fuels_abigen_macro::abigen;
+use fuels_contract::contract::Contract;
+use rand::rngs::StdRng;
+use rand::{Rng, SeedableRng};
+
+// Generate Rust bindings from our contract JSON ABI
+// FIXME: Incorrect path, see https://github.com/FuelLabs/fuels-rs/issues/94
+abigen!(MyContract, "./examples/hello_world/my-contract-abi.json");
```

Comment:
We might need to resolve https://github.com/FuelLabs/fuels-rs/issues/94 before landing this, as this path might be confusing for newcomers reading through the book.